ext/rugged/rugged_repo.c

```diff
@@ -2710,6 +2710,29 @@ static VALUE rb_git_repo_cherrypick_commit(int argc, VALUE *argv, VALUE self)
 	return rugged_index_new(rb_cRuggedIndex, self, index);
 }
 
+/*
+ *  call-seq: repo.diff_from_buffer(buffer) -> Rugged::Diff object
+ *  
+ *  Where +buffer+ is a +String+.
+ *  Returns A Rugged::Diff object
+ */
+static VALUE rb_git_diff_from_buffer(VALUE self, VALUE rb_buffer)
+{
+  git_diff *diff = NULL;
```

Comment:
Code looks good but indentation is wrong again :sweat_smile:

Reply:
Sorry about that. It looked the same in my instance of TextMate, but I should've checked more closely. Should be fixed.